bug: transformers vs 1.3.0 "TypeError: __init__() got an unexpected keyword argument 'task'"

### Describe the bug

The 1.3.0 doesn't work with recent version of hugginface `transformers`, 1.2.19 is ok.

```
2024-07-23T18:18:57+0200 [ERROR] [service:TransformersSegmentationModel:1] Initializing service error
Traceback (most recent call last):
  File "/home/matej/local/conda/envs/pose_service/lib/python3.9/site-packages/_bentoml_sdk/service/factory.py", line 257, in __call__
    instance = self.inner()
  File "/home/matej/prace/egoscue/mmpose_service/src/pose_service/segmentation.py", line 21, in __init__
    self.extractor_model = self.extractor_model_ref.load_model()
  File "/home/matej/local/conda/envs/pose_service/lib/python3.9/site-packages/bentoml/_internal/models/model.py", line 377, in load_model
    self._model = self.info.imported_module.load_model(self, *args, **kwargs)
  File "/home/matej/local/conda/envs/pose_service/lib/python3.9/site-packages/bentoml/_internal/frameworks/transformers.py", line 402, in load_model
    options = t.cast(TransformersOptions, bento_model.info.options)
  File "/home/matej/local/conda/envs/pose_service/lib/python3.9/site-packages/bentoml/_internal/models/model.py", line 635, in options
    self.imported_module.ModelOptions(**self._options),
TypeError: __init__() got an unexpected keyword argument 'task'
```

I can add more info if necessary. 

### To reproduce

_No response_

### Expected behavior

_No response_

### Environment

#### Environment variable

```bash
BENTOML_DEBUG=''
BENTOML_QUIET=''
BENTOML_BUNDLE_LOCAL_BUILD=''
BENTOML_DO_NOT_TRACK=''
BENTOML_CONFIG=''
BENTOML_CONFIG_OPTIONS=''
BENTOML_PORT=''
BENTOML_HOST=''
BENTOML_API_WORKERS=''
```

#### System information

`bentoml`: 1.3.0
`python`: 3.9.18
`platform`: Linux-6.9.7-200.fc40.x86_64-x86_64-with-glibc2.39
`uid_gid`: 1000:1000
`conda`: 22.9.0
`in_conda_env`: True
<details><summary><code>conda_packages</code></summary>

<br>

```yaml
name: pose_service
channels:
  - conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=2_kmp_llvm
  - argon2-cffi=21.3.0=pyhd3eb1b0_0
  - argon2-cffi-bindings=21.2.0=py39h7f8727e_0
  - asttokens=2.0.5=pyhd3eb1b0_0
  - async-lru=2.0.4=py39h06a4308_0
  - attrs=23.1.0=py39h06a4308_0
  - aws-c-auth=0.7.11=h0b4cabd_1
  - aws-c-cal=0.6.9=h14ec70c_3
  - aws-c-common=0.9.12=hd590300_0
  - aws-c-compression=0.2.17=h572eabf_8
  - aws-c-event-stream=0.4.1=h97bb272_2
  - aws-c-http=0.8.0=h9129f04_2
  - aws-c-io=0.14.0=hf8f278a_1
  - aws-c-mqtt=0.10.1=h2b97f5f_0
  - aws-c-s3=0.4.9=hca09fc5_0
  - aws-c-sdkutils=0.1.13=h572eabf_1
  - aws-checksums=0.1.17=h572eabf_7
  - aws-crt-cpp=0.26.0=h04327c0_8
  - aws-sdk-cpp=1.11.210=hba3e011_10
  - babel=2.11.0=py39h06a4308_0
  - backcall=0.2.0=pyhd3eb1b0_0
  - beautifulsoup4=4.12.2=py39h06a4308_0
  - bleach=4.1.0=pyhd3eb1b0_0
  - brotli-python=1.0.9=py39h6a678d5_7
  - bzip2=1.0.8=hd590300_5
  - c-ares=1.25.0=hd590300_0
  - ca-certificates=2023.11.17=hbcca054_0
  - certifi=2023.11.17=pyhd8ed1ab_0
  - cffi=1.16.0=py39h5eee18b_0
  - comm=0.1.2=py39h06a4308_0
  - cryptography=41.0.7=py39hdda0065_0
  - cyrus-sasl=2.1.28=h52b45da_1
  - dbus=1.13.18=hb2f20db_0
  - debugpy=1.6.7=py39h6a678d5_0
  - decorator=5.1.1=pyhd3eb1b0_0
  - defusedxml=0.7.1=pyhd3eb1b0_0
  - executing=0.8.3=pyhd3eb1b0_0
  - expat=2.5.0=h6a678d5_0
  - fontconfig=2.14.1=h4c34cd2_2
  - freetype=2.12.1=h4a9f257_0
  - gflags=2.2.2=he1b5a44_1004
  - giflib=5.2.1=h0b41bf4_3
  - glib=2.69.1=he621ea3_2
  - glog=0.6.0=h6f12383_0
  - gst-plugins-base=1.14.1=h6a678d5_1
  - gstreamer=1.14.1=h5eee18b_1
  - icu=73.1=h6a678d5_0
  - idna=3.4=py39h06a4308_0
  - importlib_metadata=7.0.1=hd3eb1b0_0
  - ipykernel=6.28.0=py39h06a4308_0
  - ipython=8.15.0=py39h06a4308_0
  - ipywidgets=8.0.4=py39h06a4308_0
  - jedi=0.18.1=py39h06a4308_1
  - jinja2=3.1.2=py39h06a4308_0
  - jpeg=9e=h5eee18b_1
  - json5=0.9.6=pyhd3eb1b0_0
  - jupyter=1.0.0=py39h06a4308_8
  - jupyter-lsp=2.2.0=py39h06a4308_0
  - jupyter_client=8.6.0=py39h06a4308_0
  - jupyter_console=6.6.3=py39h06a4308_0
  - jupyter_core=5.5.0=py39h06a4308_0
  - jupyter_events=0.8.0=py39h06a4308_0
  - jupyter_server=2.10.0=py39h06a4308_0
  - jupyter_server_terminals=0.4.4=py39h06a4308_1
  - jupyterlab_pygments=0.1.2=py_0
  - jupyterlab_server=2.25.1=py39h06a4308_0
  - jupyterlab_widgets=3.0.9=py39h06a4308_0
  - krb5=1.20.1=h143b758_1
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.38=h1181459_1
  - lerc=3.0=h295c915_0
  - libabseil=20230802.1=cxx17_h59595ed_0
  - libarrow=14.0.2=h84dd17c_2_cpu
  - libarrow-acero=14.0.2=h59595ed_2_cpu
  - libarrow-dataset=14.0.2=h59595ed_2_cpu
  - libarrow-flight=14.0.2=h120cb0d_2_cpu
  - libarrow-flight-sql=14.0.2=h61ff412_2_cpu
  - libarrow-gandiva=14.0.2=hacb8726_2_cpu
  - libarrow-substrait=14.0.2=h61ff412_2_cpu
  - libblas=3.9.0=21_linux64_openblas
  - libbrotlicommon=1.1.0=hd590300_1
  - libbrotlidec=1.1.0=hd590300_1
  - libbrotlienc=1.1.0=hd590300_1
  - libcblas=3.9.0=21_linux64_openblas
  - libclang13=14.0.6=default_he11475f_1
  - libcrc32c=1.1.2=h9c3ff4c_0
  - libcups=2.4.2=h2d74bed_1
  - libcurl=8.5.0=h251f7ec_0
  - libdeflate=1.17=h5eee18b_1
  - libedit=3.1.20230828=h5eee18b_0
  - libev=4.33=hd590300_2
  - libevent=2.1.12=hf998b51_1
  - libffi=3.4.4=h6a678d5_0
  - libgcc-ng=13.2.0=h807b86a_3
  - libgfortran-ng=13.2.0=h69a702a_3
  - libgfortran5=13.2.0=ha4646dd_3
  - libgoogle-cloud=2.12.0=h5206363_4
  - libgrpc=1.59.3=hd6c4280_0
  - liblapack=3.9.0=21_linux64_openblas
  - libllvm14=14.0.6=hdb19cb5_3
  - libllvm15=15.0.7=hadd5161_1
  - libnghttp2=1.58.0=h47da74e_1
  - libnl=3.9.0=hd590300_0
  - libnuma=2.0.16=h0b41bf4_1
  - libopenblas=0.3.26=pthreads_h413a1c8_0
  - libparquet=14.0.2=h352af49_2_cpu
  - libpng=1.6.39=h5eee18b_0
  - libpq=12.15=hdbd6064_1
  - libprotobuf=4.24.4=hf27288f_0
  - libre2-11=2023.06.02=h7a70373_0
  - libsodium=1.0.18=h7b6447c_0
  - libssh2=1.11.0=h0841786_0
  - libstdcxx-ng=13.2.0=h7e041cc_3
  - libthrift=0.19.0=hb90f79a_1
  - libtiff=4.5.1=h6a678d5_0
  - libutf8proc=2.8.0=h166bdaf_0
  - libuuid=1.41.5=h5eee18b_0
  - libwebp=1.3.2=h11a3e52_0
  - libwebp-base=1.3.2=hd590300_0
  - libxcb=1.15=h7f8727e_0
  - libxkbcommon=1.0.1=h5eee18b_1
  - libxml2=2.10.4=hf1b16e4_1
  - libzlib=1.2.13=hd590300_5
  - llvm-openmp=17.0.6=h4dfa4b3_0
  - lz4-c=1.9.4=h6a678d5_0
  - matplotlib-inline=0.1.6=py39h06a4308_0
  - mistune=2.0.4=py39h06a4308_0
  - mysql=5.7.24=h721c034_2
  - nbclient=0.8.0=py39h06a4308_0
  - nbconvert=7.10.0=py39h06a4308_0
  - nbformat=5.9.2=py39h06a4308_0
  - ncurses=6.4=h6a678d5_0
  - nest-asyncio=1.5.6=py39h06a4308_0
  - notebook=7.0.6=py39h06a4308_0
  - notebook-shim=0.2.3=py39h06a4308_0
  - openjpeg=2.5.0=hfec8fc6_2
  - openssl=3.2.0=hd590300_1
  - orc=1.9.2=h4b38347_0
  - overrides=7.4.0=py39h06a4308_0
  - packaging=23.1=py39h06a4308_0
  - pandocfilters=1.5.0=pyhd3eb1b0_0
  - parso=0.8.3=pyhd3eb1b0_0
  - pcre=8.45=h295c915_0
  - pexpect=4.8.0=pyhd3eb1b0_3
  - pickleshare=0.7.5=pyhd3eb1b0_1003
  - ply=3.11=py39h06a4308_0
  - prometheus_client=0.14.1=py39h06a4308_0
  - prompt-toolkit=3.0.43=py39h06a4308_0
  - prompt_toolkit=3.0.43=hd3eb1b0_0
  - ptyprocess=0.7.0=pyhd3eb1b0_2
  - pure_eval=0.2.2=pyhd3eb1b0_0
  - pycparser=2.21=pyhd3eb1b0_0
  - pyopenssl=23.2.0=py39h06a4308_0
  - pyqt=5.15.10=py39h6a678d5_0
  - pyqt5-sip=12.13.0=py39h5eee18b_0
  - pysocks=1.7.1=py39h06a4308_0
  - python=3.9.18=h955ad1f_0
  - python-dateutil=2.8.2=pyhd3eb1b0_0
  - python-fastjsonschema=2.16.2=py39h06a4308_0
  - python-json-logger=2.0.7=py39h06a4308_0
  - python_abi=3.9=2_cp39
  - pyzmq=25.1.2=py39h6a678d5_0
  - qt-main=5.15.2=h53bd1ea_10
  - qtconsole=5.5.0=py39h06a4308_0
  - qtpy=2.4.1=py39h06a4308_0
  - rdma-core=49.0=hd3aeb46_2
  - re2=2023.06.02=h2873b5e_0
  - readline=8.2=h5eee18b_0
  - rerun-sdk=0.12.0=py39h3d6467e_2
  - rfc3339-validator=0.1.4=py39h06a4308_0
  - rfc3986-validator=0.1.1=py39h06a4308_0
  - s2n=1.4.1=h06160fa_0
  - send2trash=1.8.2=py39h06a4308_0
  - sip=6.7.12=py39h6a678d5_0
  - six=1.16.0=pyhd3eb1b0_1
  - snappy=1.1.10=h9fff704_0
  - sniffio=1.3.0=py39h06a4308_0
  - soupsieve=2.5=py39h06a4308_0
  - sqlite=3.41.2=h5eee18b_0
  - stack_data=0.2.0=pyhd3eb1b0_0
  - terminado=0.17.1=py39h06a4308_0
  - tinycss2=1.2.1=py39h06a4308_0
  - tk=8.6.12=h1ccaba5_0
  - tomli=2.0.1=py39h06a4308_0
  - tornado=6.3.3=py39h5eee18b_0
  - traitlets=5.7.1=py39h06a4308_0
  - ucx=1.15.0=h75e419f_3
  - webencodings=0.5.1=py39h06a4308_1
  - websocket-client=0.58.0=py39h06a4308_4
  - wheel=0.41.2=py39h06a4308_0
  - widgetsnbextension=4.0.5=py39h06a4308_0
  - xz=5.4.5=h5eee18b_0
  - yaml=0.2.5=h7b6447c_0
  - zeromq=4.3.5=h6a678d5_0
  - zlib=1.2.13=hd590300_5
  - zstd=1.5.5=hc292b87_0
  - pip:
    - absl-py==1.4.0
    - addict==2.4.0
    - aiofiles==23.1.0
    - aiohttp==3.8.4
    - aiohttp-retry==2.8.3
    - aiosignal==1.3.1
    - aiosqlite==0.20.0
    - aliyun-python-sdk-core==2.14.0
    - aliyun-python-sdk-kms==2.16.2
    - amqp==5.1.1
    - annotated-types==0.6.0
    - antlr4-python3-runtime==4.9.3
    - anyio==4.4.0
    - appdirs==1.4.4
    - argcomplete==3.1.1
    - array-record==0.5.0
    - arrow==1.3.0
    - asgiref==3.6.0
    - astunparse==1.6.3
    - async-timeout==4.0.2
    - asyncssh==2.14.0
    - atpublic==4.0
    - av==9.2.0
    - backoff==2.2.1
    - bentoml==1.3.0
    - billiard==4.1.0
    - blinker==1.7.0
    - boto3==1.26.109
    - botocore==1.29.109
    - build==1.1.1
    - bump-pydantic==0.8.0
    - cachetools==5.3.1
    - cattrs==22.2.0
    - celery==5.3.4
    - charset-normalizer==2.1.1
    - chex==0.1.7
    - chumpy==0.70
    - circus==0.18.0
    - click==8.1.3
    - click-didyoumean==0.3.0
    - click-option-group==0.5.5
    - click-plugins==1.1.1
    - click-repl==0.3.0
    - cloudpickle==2.2.1
    - colorama==0.4.6
    - configargparse==1.7
    - configobj==5.0.8
    - contextlib2==21.6.0
    - contourpy==1.0.7
    - coverage==7.3.2
    - crcmod==1.7
    - cuda-python==12.3.0
    - cycler==0.11.0
    - cython==0.29.34
    - dacite==1.7.0
    - dash==2.15.0
    - dash-core-components==2.0.0
    - dash-html-components==2.0.0
    - dash-table==5.0.0
    - deepmerge==1.1.0
    - deprecated==1.2.13
    - dictdiffer==0.9.0
    - dill==0.3.6
    - diskcache==5.6.3
    - distlib==0.3.8
    - distro==1.8.0
    - dm-tree==0.1.8
    - dnspython==2.3.0
    - docker-pycreds==0.4.0
    - dpath==2.1.6
    - dulwich==0.21.6
    - dvc==3.28.0
    - dvc-data==2.20.0
    - dvc-http==2.30.2
    - dvc-objects==1.1.0
    - dvc-render==0.6.0
    - dvc-studio-client==0.15.0
    - dvc-task==0.3.0
    - editables==0.5
    - et-xmlfile==1.1.0
    - etils==1.4.1
    - eventlet==0.33.3
    - exceptiongroup==1.1.1
    - fiftyone==0.23.8
    - fiftyone-brain==0.16.1
    - fiftyone-db==0.4.0
    - fiftyone-desktop==0.33.7
    - filelock==3.13.1
    - flake8==6.1.0
    - flask==3.0.0
    - flask-basicauth==0.2.0
    - flask-cors==4.0.0
    - flask-login==0.6.3
    - flatbuffers==23.5.26
    - flatten-dict==0.4.2
    - flax==0.7.4
    - flufl-lock==7.1.1
    - fonttools==4.39.3
    - fqdn==1.5.1
    - frozenlist==1.3.3
    - fs==2.4.16
    - fs-s3fs==1.1.1
    - fsspec==2023.10.0
    - ftfy==6.1.1
    - funcy==2.0
    - future==0.18.3
    - gast==0.4.0
    - gevent==23.9.1
    - geventhttpclient==2.3.1
    - gin-config==0.5.0
    - gitdb==4.0.11
    - gitpython==3.1.40
    - glob2==0.7
    - google-api-core==2.18.0
    - google-api-python-client==2.125.0
    - google-auth==2.22.0
    - google-auth-httplib2==0.2.0
    - google-auth-oauthlib==1.0.0
    - google-pasta==0.2.0
    - googleapis-common-protos==1.59.0
    - grandalf==0.8
    - graphql-core==3.2.3
    - greenlet==2.0.2
    - grpcio==1.54.2
    - grpcio-health-checking==1.48.2
    - gto==1.5.0
    - h11==0.14.0
    - h2==4.1.0
    - h5py==3.10.0
    - hatch==1.9.3
    - hatchling==1.21.1
    - hpack==4.0.0
    - httpcore==1.0.5
    - httplib2==0.22.0
    - httpx==0.27.0
    - httpx-ws==0.6.0
    - huggingface-hub==0.24.1
    - humanize==4.8.0
    - hydra-core==1.3.2
    - hypercorn==0.14.3
    - hyperframe==6.0.1
    - hyperlink==21.0.0
    - imageio==2.28.1
    - imgaug==0.4.0
    - immutabledict==4.2.0
    - importlib-metadata==6.2.0
    - importlib-resources==5.12.0
    - inflate64==0.3.1
    - inflection==0.5.1
    - iniconfig==2.0.0
    - inquirerpy==0.3.4
    - interrogate==1.5.0
    - ipympl==0.9.3
    - ipython-genutils==0.2.0
    - isoduration==20.11.0
    - isort==4.3.21
    - iterative-telemetry==0.0.8
    - itsdangerous==2.1.2
    - jaraco-classes==3.3.1
    - jax==0.4.14
    - jaxlib==0.4.14
    - jeepney==0.8.0
    - jmespath==0.10.0
    - joblib==1.2.0
    - json-stream==2.3.2
    - json-stream-rs-tokenizer==0.4.25
    - json-tricks==3.16.1
    - jsonlines==3.1.0
    - jsonpointer==2.4
    - jsonschema==4.20.0
    - jsonschema-specifications==2023.11.1
    - jstyleson==0.0.2
    - jupyter-events==0.9.0
    - jupyter-server==2.10.1
    - jupyterlab==4.0.9
    - kaggle==1.6.11
    - kaleido==0.2.1
    - keras==2.15.0
    - keyring==24.3.1
    - kiwisolver==1.4.4
    - kombu==5.3.2
    - kornia==0.7.2
    - kornia-moons==0.2.9
    - kornia-rs==0.1.3
    - lazy-loader==0.2
    - libclang==16.0.6
    - libcst==1.1.0
    - llvmlite==0.41.1
    - locust==2.29.1
    - lxml==5.2.1
    - markdown==3.4.3
    - markdown-it-py==2.2.0
    - markupsafe==2.1.2
    - matplotlib==3.8.0
    - mccabe==0.7.0
    - mdurl==0.1.2
    - mediapipe==0.10.3
    - ml-dtypes==0.3.2
    - mmcls==0.25.0
    - mmcv==2.1.0
    - mmdet==3.2.0
    - mmsegmentation==0.30.0
    - model-index==0.1.11
    - mongoengine==0.24.2
    - more-itertools==10.2.0
    - motor==3.1.2
    - mpmath==1.2.1
    - msgpack==1.0.5
    - multidict==6.0.4
    - multivolumefile==0.2.3
    - munkres==1.1.4
    - mypy-extensions==1.0.0
    - namex==0.0.7
    - networkx==2.8.8
    - nibabel==5.1.0
    - numba==0.58.1
    - numpy==1.26.4
    - nvidia-ml-py==11.525.150
    - oauth2client==4.1.3
    - oauthlib==3.2.2
    - omegaconf==2.3.0
    - open3d==0.18.0
    - open3d-cpu==0.18.0
    - opencv-contrib-python==4.9.0.80
    - opendatalab==0.0.10
    - openmim==0.3.9
    - openpyxl==3.1.2
    - opentelemetry-api==1.20.0
    - opentelemetry-exporter-otlp-proto-http==1.14.0
    - opentelemetry-instrumentation==0.41b0
    - opentelemetry-instrumentation-aiohttp-client==0.41b0
    - opentelemetry-instrumentation-asgi==0.41b0
    - opentelemetry-instrumentation-grpc==0.38b0
    - opentelemetry-proto==1.14.0
    - opentelemetry-sdk==1.20.0
    - opentelemetry-semantic-conventions==0.41b0
    - opentelemetry-util-http==0.41b0
    - openvino==2023.1.0
    - openvino-dev==2023.1.0
    - openvino-telemetry==2023.2.1
    - openxlab==0.0.26
    - opt-einsum==3.3.0
    - optax==0.1.7
    - optree==0.11.0
    - orbax-checkpoint==0.3.5
    - ordered-set==4.1.0
    - orjson==3.9.1
    - oss2==2.17.0
    - ovmsclient==2023.1
    - pandas==2.0.0
    - parameterized==0.9.0
    - pathspec==0.11.1
    - pfzy==0.3.4
    - pillow==9.5.0
    - pillow-heif==0.10.1
    - pip==24.0
    - pip-requirements-parser==32.0.1
    - pip-tools==7.4.0
    - pipdeptree==2.19.0
    - platformdirs==3.11.0
    - plotly==5.15.0
    - pluggy==1.5.0
    - plyfile==0.9
    - portalocker==2.8.2
    - pose-service==0.0.1
    - pprintpp==0.4.0
    - prettytable==3.6.0
    - priority==2.0.0
    - prometheus-client==0.16.0
    - promise==2.3
    - proto-plus==1.23.0
    - protobuf==3.20.3
    - psutil==5.9.4
    - py==1.11.0
    - py-cpuinfo==9.0.0
    - py7zr==0.20.5
    - pyarrow==14.0.2
    - pyasn1==0.5.0
    - pyasn1-modules==0.3.0
    - pybcj==1.0.1
    - pycocotools==2.0.6
    - pycodestyle==2.11.1
    - pycryptodome==3.19.0
    - pycryptodomex==3.18.0
    - pydantic==2.7.1
    - pydantic-core==2.18.2
    - pydot==1.4.2
    - pyemd==1.0.0
    - pyflakes==3.1.0
    - pygit2==1.13.2
    - pygments==2.16.1
    - pygtrie==2.5.0
    - pyinquirer==1.0.3
    - pymongo==4.3.3
    - pynndescent==0.5.11
    - pynvml==11.5.0
    - pyodbc==5.0.1
    - pyparsing==3.0.9
    - pyppmd==1.0.0
    - pyproject-hooks==1.0.0
    - pyquaternion==0.9.9
    - pyrealsense2==2.54.2.5684
    - pytest==8.2.2
    - pytest-runner==6.0.0
    - python-dotenv==1.0.1
    - python-multipart==0.0.6
    - python-rapidjson==1.12
    - python-slugify==8.0.4
    - pytz==2023.3
    - pywavelets==1.4.1
    - pyyaml==6.0
    - pyzstd==0.15.8
    - rarfile==4.0
    - referencing==0.33.0
    - regex==2023.3.23
    - requests==2.28.2
    - requests-oauthlib==1.3.1
    - requests-toolbelt==1.0.0
    - retrying==1.3.4
    - rich==13.4.2
    - roundrobin==0.0.4
    - rpds-py==0.13.1
    - rsa==4.9
    - ruamel-yaml==0.17.32
    - ruamel-yaml-clib==0.2.7
    - s3transfer==0.6.0
    - sacrebleu==2.4.1
    - safetensors==0.4.3
    - schema==0.7.5
    - scikit-image==0.21.0
    - scikit-learn==1.1.2
    - scipy==1.10.1
    - scmrepo==1.4.1
    - seaborn==0.13.2
    - secretstorage==3.3.3
    - semver==3.0.2
    - sentencepiece==0.2.0
    - sentry-sdk==1.45.0
    - seqeval==1.2.2
    - setproctitle==1.3.3
    - setuptools==60.2.0
    - shapely==2.0.1
    - shellingham==1.5.4
    - shortuuid==1.0.11
    - shtab==1.6.4
    - simple-di==0.1.5
    - smmap==5.0.1
    - sortedcontainers==2.4.0
    - sounddevice==0.4.6
    - sqltrie==0.8.0
    - sse-starlette==0.10.3
    - sseclient-py==1.7.2
    - starlette==0.25.0
    - strawberry-graphql==0.138.1
    - sympy==1.11.1
    - tabulate==0.9.0
    - tenacity==8.2.2
    - tensorboard==2.15.2
    - tensorboard-data-server==0.7.1
    - tensorboardx==2.6.2.2
    - tensorflow==2.15.1
    - tensorflow-addons==0.23.0
    - tensorflow-datasets==4.9.3
    - tensorflow-decision-forests==1.8.1
    - tensorflow-estimator==2.15.0
    - tensorflow-hub==0.16.1
    - tensorflow-io-gcs-filesystem==0.34.0
    - tensorflow-metadata==1.14.0
    - tensorflow-model-optimization==0.8.0
    - tensorflowjs==4.18.0
    - tensorstore==0.1.43
    - termcolor==2.3.0
    - terminaltables==3.1.10
    - text-unidecode==1.3
    - texttable==1.6.7
    - tf-keras==2.15.1
    - tf-slim==1.1.0
    - threadpoolctl==3.1.0
    - tifffile==2023.4.12
    - tokenizers==0.19.1
    - toml==0.10.2
    - tomli-w==1.0.0
    - tomlkit==0.12.2
    - toolz==0.12.0
    - torch==2.0.0+cpu
    - torchvision==0.15.1+cpu
    - tqdm==4.65.0
    - transformers==4.43.1
    - tritonclient==2.38.0
    - trove-classifiers==2024.2.23
    - typeguard==2.13.3
    - typer==0.9.0
    - types-python-dateutil==2.8.19.14
    - typing-extensions==4.11.0
    - typing-inspect==0.9.0
    - tzdata==2023.3
    - tzlocal==5.0.1
    - umap-learn==0.5.5
    - universal-analytics-python3==1.1.1
    - uri-template==1.3.0
    - uritemplate==4.1.1
    - urllib3==1.26.13
    - userpath==1.9.2
    - uv==0.2.27
    - uvicorn==0.30.1
    - vine==5.0.0
    - virtualenv==20.25.1
    - voluptuous==0.13.1
    - voxel51-eta==0.12.6
    - wandb==0.16.6
    - watchfiles==0.19.0
    - wcwidth==0.2.6
    - webcolors==1.13
    - werkzeug==3.0.1
    - wrapt==1.14.1
    - wsproto==1.2.0
    - wurlitzer==3.0.3
    - xdoctest==1.1.2
    - xmltodict==0.13.0
    - xtcocotools==1.13
    - yapf==0.32.0
    - yarl==1.8.2
    - zc-lockfile==3.0.post1
    - zipp==3.15.0
    - zope-event==5.0
    - zope-interface==6.1
    - zstandard==0.22.0
prefix: /home/matej/local/conda/envs/pose_service
```

</details>

<details><summary><code>pip_packages</code></summary>

<br>

```
absl-py==1.4.0
addict==2.4.0
aiofiles==23.1.0
aiohttp==3.8.4
aiohttp-retry==2.8.3
aiosignal==1.3.1
aiosqlite==0.20.0
aliyun-python-sdk-core==2.14.0
aliyun-python-sdk-kms==2.16.2
amqp==5.1.1
annotated-types==0.6.0
antlr4-python3-runtime==4.9.3
anyio==4.4.0
appdirs==1.4.4
argcomplete==3.1.1
argon2-cffi @ file:///opt/conda/conda-bld/argon2-cffi_1645000214183/work
argon2-cffi-bindings @ file:///tmp/build/80754af9/argon2-cffi-bindings_1644569679365/work
array-record==0.5.0
arrow==1.3.0
asgiref==3.6.0
asttokens @ file:///opt/conda/conda-bld/asttokens_1646925590279/work
astunparse==1.6.3
async-lru @ file:///croot/async-lru_1699554519285/work
async-timeout==4.0.2
asyncssh==2.14.0
atpublic==4.0
attrs @ file:///croot/attrs_1695717823297/work
av==9.2.0
babel @ file:///croot/babel_1671781930836/work
backcall @ file:///home/ktietz/src/ci/backcall_1611930011877/work
backoff==2.2.1
beautifulsoup4 @ file:///croot/beautifulsoup4-split_1681493039619/work
bentoml==1.3.0
billiard==4.1.0
bleach @ file:///opt/conda/conda-bld/bleach_1641577558959/work
blinker==1.7.0
boto3==1.26.109
botocore==1.29.109
brotli @ file:///tmp/abs_ecyw11_7ze/croots/recipe/brotli-split_1659616059936/work
build==1.1.1
bump-pydantic==0.8.0
cachetools==5.3.1
cattrs==22.2.0
celery==5.3.4
certifi @ file:///home/conda/feedstock_root/build_artifacts/certifi_1700303426725/work/certifi
cffi @ file:///croot/cffi_1700254295673/work
charset-normalizer @ file:///tmp/build/80754af9/charset-normalizer_1630003229654/work
charset-normalizer==2.1.1
chex==0.1.7
chumpy==0.70
circus==0.18.0
click==8.1.3
click-didyoumean==0.3.0
click-option-group==0.5.5
click-plugins==1.1.1
click-repl==0.3.0
cloudpickle==2.2.1
colorama==0.4.6
comm @ file:///croot/comm_1671231121260/work
configargparse==1.7
configobj==5.0.8
contextlib2==21.6.0
contourpy==1.0.7
coverage==7.3.2
crcmod==1.7
cryptography @ file:///croot/cryptography_1702070282333/work
cuda-python==12.3.0
cycler==0.11.0
cython==0.29.34
dacite==1.7.0
dash==2.15.0
dash-core-components==2.0.0
dash-html-components==2.0.0
dash-table==5.0.0
debugpy @ file:///croot/debugpy_1690905042057/work
decorator @ file:///opt/conda/conda-bld/decorator_1643638310831/work
deepmerge==1.1.0
defusedxml @ file:///tmp/build/80754af9/defusedxml_1615228127516/work
deprecated==1.2.13
dictdiffer==0.9.0
dill==0.3.6
diskcache==5.6.3
distlib==0.3.8
distro==1.8.0
dm-tree==0.1.8
dnspython==2.3.0
docker-pycreds==0.4.0
dpath==2.1.6
dulwich==0.21.6
dvc==3.28.0
dvc-data==2.20.0
dvc-http==2.30.2
dvc-objects==1.1.0
dvc-render==0.6.0
dvc-studio-client==0.15.0
dvc-task==0.3.0
editables==0.5
et-xmlfile==1.1.0
etils==1.4.1
eventlet==0.33.3
exceptiongroup==1.1.1
exceptiongroup @ file:///croot/exceptiongroup_1706031385326/work
executing @ file:///opt/conda/conda-bld/executing_1646925071911/work
fastjsonschema @ file:///opt/conda/conda-bld/python-fastjsonschema_1661371079312/work
fiftyone==0.23.8
fiftyone-brain==0.16.1
fiftyone-db==0.4.0
fiftyone-desktop==0.33.7
filelock==3.13.1
flake8==6.1.0
flask==3.0.0
flask-basicauth==0.2.0
flask-cors==4.0.0
flask-login==0.6.3
flatbuffers==23.5.26
flatten-dict==0.4.2
flax==0.7.4
flufl-lock==7.1.1
fonttools==4.39.3
fqdn==1.5.1
frozenlist==1.3.3
fs==2.4.16
fs-s3fs==1.1.1
fsspec==2023.10.0
ftfy==6.1.1
funcy==2.0
future==0.18.3
gast==0.4.0
gevent==23.9.1
geventhttpclient==2.3.1
gin-config==0.5.0
gitdb==4.0.11
gitpython==3.1.40
glob2==0.7
google-api-core==2.18.0
google-api-python-client==2.125.0
google-auth==2.22.0
google-auth-httplib2==0.2.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
googleapis-common-protos==1.59.0
grandalf==0.8
graphql-core==3.2.3
greenlet==2.0.2
grpcio==1.54.2
grpcio-health-checking==1.48.2
gto==1.5.0
h11==0.14.0
h2==4.1.0
h5py==3.10.0
hatch==1.9.3
hatchling==1.21.1
hpack==4.0.0
httpcore==1.0.5
httplib2==0.22.0
httpx==0.27.0
httpx-ws==0.6.0
huggingface-hub==0.24.1
humanize==4.8.0
hydra-core==1.3.2
hypercorn==0.14.3
hyperframe==6.0.1
hyperlink==21.0.0
idna @ file:///croot/idna_1666125576474/work
imageio==2.28.1
imgaug==0.4.0
immutabledict==4.2.0
importlib-metadata==6.2.0
importlib-metadata @ file:///croot/importlib_metadata-suite_1704813515092/work
importlib-resources==5.12.0
inflate64==0.3.1
inflection==0.5.1
iniconfig==2.0.0
inquirerpy==0.3.4
interrogate==1.5.0
ipykernel @ file:///croot/ipykernel_1705933831282/work
ipympl==0.9.3
ipython @ file:///croot/ipython_1694181358621/work
ipython-genutils==0.2.0
ipywidgets @ file:///croot/ipywidgets_1679394798311/work
isoduration==20.11.0
isort==4.3.21
iterative-telemetry==0.0.8
itsdangerous==2.1.2
jaraco-classes==3.3.1
jax==0.4.14
jaxlib==0.4.14
jedi @ file:///tmp/build/80754af9/jedi_1644297102865/work
jeepney==0.8.0
jinja2 @ file:///croot/jinja2_1666908132255/work
jmespath==0.10.0
joblib==1.2.0
json-stream==2.3.2
json-stream-rs-tokenizer==0.4.25
json-tricks==3.16.1
json5 @ file:///tmp/build/80754af9/json5_1624432770122/work
jsonlines==3.1.0
jsonpointer==2.4
jsonschema @ file:///croot/jsonschema_1699041609003/work
jsonschema==4.20.0
jsonschema-specifications @ file:///croot/jsonschema-specifications_1699032386549/work
jsonschema-specifications==2023.11.1
jstyleson==0.0.2
jupyter @ file:///tmp/abs_33h4eoipez/croots/recipe/jupyter_1659349046347/work
jupyter-client @ file:///croot/jupyter_client_1699455897726/work
jupyter-console @ file:///croot/jupyter_console_1679999630278/work
jupyter-core @ file:///croot/jupyter_core_1698937308754/work
jupyter-events @ file:///croot/jupyter_events_1699282461638/work
jupyter-events==0.9.0
jupyter-lsp @ file:///croot/jupyter-lsp-meta_1699978238815/work
jupyter-server @ file:///croot/jupyter_server_1699466442171/work
jupyter-server==2.10.1
jupyter-server-terminals @ file:///croot/jupyter_server_terminals_1686870725608/work
jupyterlab @ file:///croot/jupyterlab_1700518272623/work
jupyterlab==4.0.9
jupyterlab-pygments @ file:///tmp/build/80754af9/jupyterlab_pygments_1601490720602/work
jupyterlab-server @ file:///croot/jupyterlab_server_1699555425460/work
jupyterlab-widgets @ file:///croot/jupyterlab_widgets_1700168618520/work
kaggle==1.6.11
kaleido==0.2.1
keras==2.15.0
keyring==24.3.1
kiwisolver==1.4.4
kombu==5.3.2
kornia==0.7.2
kornia-moons==0.2.9
kornia-rs==0.1.3
lazy-loader==0.2
libclang==16.0.6
libcst==1.1.0
llvmlite==0.41.1
locust==2.29.1
lxml==5.2.1
markdown==3.4.3
markdown-it-py==2.2.0
markupsafe==2.1.2
markupsafe @ file:///croot/markupsafe_1704205993651/work
matplotlib==3.8.0
matplotlib-inline @ file:///opt/conda/conda-bld/matplotlib-inline_1662014470464/work
mccabe==0.7.0
mdurl==0.1.2
mediapipe==0.10.3
mistune @ file:///opt/conda/conda-bld/mistune_1661496219659/work
ml-dtypes==0.3.2
mmcls==0.25.0
mmcv==2.1.0
mmdet==3.2.0
-e /home/matej/prace/egoscue/3rd_party/mmengine
-e /home/matej/prace/egoscue/mmpose
mmsegmentation==0.30.0
model-index==0.1.11
mongoengine==0.24.2
more-itertools==10.2.0
motor==3.1.2
mpmath==1.2.1
msgpack==1.0.5
multidict==6.0.4
multivolumefile==0.2.3
munkres==1.1.4
mypy-extensions==1.0.0
namex==0.0.7
nbclient @ file:///croot/nbclient_1698934205032/work
nbconvert @ file:///croot/nbconvert_1699022732553/work
nbformat @ file:///croot/nbformat_1694616755618/work
nest-asyncio @ file:///croot/nest-asyncio_1672387112409/work
networkx==2.8.8
nibabel==5.1.0
notebook @ file:///croot/notebook_1700582094678/work
notebook-shim @ file:///croot/notebook-shim_1699455894279/work
numba==0.58.1
numpy==1.24.3
numpy==1.26.4
nvidia-ml-py==11.525.150
oauth2client==4.1.3
oauthlib==3.2.2
omegaconf==2.3.0
open3d==0.18.0
open3d-cpu==0.18.0
opencv-contrib-python==4.9.0.80
opendatalab==0.0.10
openmim==0.3.9
openpyxl==3.1.2
opentelemetry-api==1.20.0
opentelemetry-exporter-otlp-proto-http==1.14.0
opentelemetry-instrumentation==0.41b0
opentelemetry-instrumentation-aiohttp-client==0.41b0
opentelemetry-instrumentation-asgi==0.41b0
opentelemetry-instrumentation-grpc==0.38b0
opentelemetry-proto==1.14.0
opentelemetry-sdk==1.20.0
opentelemetry-semantic-conventions==0.41b0
opentelemetry-util-http==0.41b0
openvino==2023.1.0
openvino-dev==2023.1.0
openvino-telemetry==2023.2.1
openxlab==0.0.26
opt-einsum==3.3.0
optax==0.1.7
optree==0.11.0
orbax-checkpoint==0.3.5
ordered-set==4.1.0
orjson==3.9.1
oss2==2.17.0
overrides @ file:///croot/overrides_1699371140756/work
ovmsclient==2023.1
packaging @ file:///croot/packaging_1693575174725/work
pandas==2.0.0
pandocfilters @ file:///opt/conda/conda-bld/pandocfilters_1643405455980/work
parameterized==0.9.0
parso @ file:///opt/conda/conda-bld/parso_1641458642106/work
pathspec==0.11.1
pexpect @ file:///tmp/build/80754af9/pexpect_1605563209008/work
pfzy==0.3.4
pickleshare @ file:///tmp/build/80754af9/pickleshare_1606932040724/work
pillow==9.5.0
pillow @ file:///croot/pillow_1696580024257/work
pillow-heif==0.10.1
pip==24.0
pip-requirements-parser==32.0.1
pip-tools==7.4.0
pipdeptree==2.19.0
-e /home/matej/prace/equiscore/3rd_party/pixelstitch
platformdirs @ file:///croot/platformdirs_1692205439124/work
platformdirs==3.11.0
plotly==5.15.0
pluggy==1.5.0
ply==3.11
plyfile==0.9
portalocker==2.8.2
-e file:///home/matej/prace/egoscue/mmpose_service
pprintpp==0.4.0
prettytable==3.6.0
priority==2.0.0
prometheus-client @ file:///tmp/abs_d3zeliano1/croots/recipe/prometheus_client_1659455100375/work
prometheus-client==0.16.0
promise==2.3
prompt-toolkit==3.0.43
proto-plus==1.23.0
protobuf==3.20.3
psutil==5.9.4
psutil==6.0.0
ptyprocess @ file:///tmp/build/80754af9/ptyprocess_1609355006118/work/dist/ptyprocess-0.7.0-py2.py3-none-any.whl
pure-eval @ file:///opt/conda/conda-bld/pure_eval_1646925070566/work
py==1.11.0
py-cpuinfo==9.0.0
py7zr==0.20.5
pyarrow==14.0.2
pyarrow==14.0.2
pyasn1==0.5.0
pyasn1-modules==0.3.0
pybcj==1.0.1
pycocotools==2.0.6
pycodestyle==2.11.1
pycparser @ file:///tmp/build/80754af9/pycparser_1636541352034/work
pycryptodome==3.19.0
pycryptodomex==3.18.0
pydantic==2.7.1
pydantic-core==2.18.2
pydot==1.4.2
pyemd==1.0.0
pyflakes==3.1.0
pygit2==1.13.2
pygments==2.14.0
pygments @ file:///croot/pygments_1684279966437/work
pygments==2.16.1
pygtrie==2.5.0
pyinquirer==1.0.3
pymongo==4.3.3
pynndescent==0.5.11
pynvml==11.5.0
pyodbc==5.0.1
pyopenssl @ file:///croot/pyopenssl_1690223430423/work
pyparsing==3.0.9
pyppmd==1.0.0
pyproject-hooks==1.0.0
pyqt5==5.15.10
pyqt5-sip @ file:///croot/pyqt-split_1698769088074/work/pyqt_sip
pyquaternion==0.9.9
pyrealsense2==2.54.2.5684
pysocks @ file:///tmp/build/80754af9/pysocks_1605305812635/work
pytest==8.2.2
pytest-runner==6.0.0
python-dateutil @ file:///tmp/build/80754af9/python-dateutil_1626374649649/work
python-dotenv==1.0.1
python-json-logger @ file:///croot/python-json-logger_1683823803357/work
python-multipart==0.0.6
python-rapidjson==1.12
python-slugify==8.0.4
pytz==2023.3
pytz @ file:///croot/pytz_1695131579487/work
pywavelets==1.4.1
pyyaml==6.0
pyyaml @ file:///croot/pyyaml_1698096049011/work
pyzmq @ file:///croot/pyzmq_1705605076900/work
pyzstd==0.15.8
qtconsole @ file:///croot/qtconsole_1700160644874/work
qtpy @ file:///croot/qtpy_1700144840038/work
rarfile==4.0
referencing==0.31.0
referencing==0.33.0
regex==2023.3.23
requests==2.28.2
requests @ file:///croot/requests_1690400202158/work
requests-oauthlib==1.3.1
requests-toolbelt==1.0.0
rerun-sdk @ file:///home/conda/feedstock_root/build_artifacts/rerun-sdk_1705065223242/work/rerun_py
retrying==1.3.4
rfc3339-validator @ file:///croot/rfc3339-validator_1683077044675/work
rfc3986-validator @ file:///croot/rfc3986-validator_1683058983515/work
rich==13.4.2
roundrobin==0.0.4
rpds-py @ file:///croot/rpds-py_1698945930462/work
rpds-py==0.13.1
rsa==4.9
ruamel-yaml==0.17.32
ruamel-yaml-clib==0.2.7
s3transfer==0.6.0
sacrebleu==2.4.1
safetensors==0.4.3
schema==0.7.5
scikit-image==0.21.0
scikit-learn==1.1.2
scipy==1.10.1
scmrepo==1.4.1
seaborn==0.13.2
secretstorage==3.3.3
semver==3.0.2
send2trash @ file:///croot/send2trash_1699371139552/work
sentencepiece==0.2.0
sentry-sdk==1.45.0
seqeval==1.2.2
setproctitle==1.3.3
setuptools==60.2.0
setuptools==68.2.2
shapely==2.0.1
shellingham==1.5.4
shortuuid==1.0.11
shtab==1.6.4
simple-di==0.1.5
sip @ file:///croot/sip_1698675935381/work
six @ file:///tmp/build/80754af9/six_1644875935023/work
smmap==5.0.1
sniffio @ file:///croot/sniffio_1705431295498/work
sortedcontainers==2.4.0
sounddevice==0.4.6
soupsieve @ file:///croot/soupsieve_1696347547217/work
sqltrie==0.8.0
sse-starlette==0.10.3
sseclient-py==1.7.2
stack-data @ file:///opt/conda/conda-bld/stack_data_1646927590127/work
starlette==0.25.0
strawberry-graphql==0.138.1
sympy==1.11.1
tabulate==0.9.0
tenacity==8.2.2
tensorboard==2.15.2
tensorboard-data-server==0.7.1
tensorboardx==2.6.2.2
tensorflow==2.15.1
tensorflow-addons==0.23.0
tensorflow-datasets==4.9.3
tensorflow-decision-forests==1.8.1
tensorflow-estimator==2.15.0
tensorflow-hub==0.16.1
tensorflow-io-gcs-filesystem==0.34.0
tensorflow-metadata==1.14.0
tensorflow-model-optimization==0.8.0
tensorflowjs==4.18.0
tensorstore==0.1.43
termcolor==2.3.0
terminado @ file:///croot/terminado_1671751832461/work
terminaltables==3.1.10
text-unidecode==1.3
texttable==1.6.7
tf-keras==2.15.1
tf-slim==1.1.0
threadpoolctl==3.1.0
tifffile==2023.4.12
tinycss2 @ file:///croot/tinycss2_1668168815555/work
tokenizers==0.19.1
toml==0.10.2
tomli @ file:///opt/conda/conda-bld/tomli_1657175507142/work
tomli-w==1.0.0
tomlkit==0.12.2
toolz==0.12.0
torch==2.0.0+cpu
torchvision==0.15.1+cpu
tornado @ file:///croot/tornado_1696936946304/work
tqdm==4.65.0
traitlets @ file:///croot/traitlets_1671143879854/work
transformers==4.43.1
tritonclient==2.38.0
trove-classifiers==2024.2.23
typeguard==2.13.3
typer==0.9.0
types-python-dateutil==2.8.19.14
typing-extensions==4.11.0
typing-inspect==0.9.0
tzdata==2023.3
tzlocal==5.0.1
umap-learn==0.5.5
universal-analytics-python3==1.1.1
uri-template==1.3.0
uritemplate==4.1.1
urllib3==1.26.13
urllib3 @ file:///croot/urllib3_1698257533958/work
userpath==1.9.2
uv==0.2.27
uvicorn==0.30.1
vine==5.0.0
virtualenv==20.25.1
voluptuous==0.13.1
voxel51-eta==0.12.6
wandb==0.16.6
watchfiles==0.19.0
wcwidth @ file:///Users/ktietz/demo/mc3/conda-bld/wcwidth_1629357192024/work
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client @ file:///tmp/build/80754af9/websocket-client_1614803975924/work
werkzeug==3.0.1
wheel==0.41.2
widgetsnbextension @ file:///croot/widgetsnbextension_1679313860248/work
wrapt==1.14.1
wsproto==1.2.0
wurlitzer==3.0.3
xdoctest==1.1.2
xmltodict==0.13.0
xtcocotools==1.13
yapf==0.32.0
yarl==1.8.2
zc-lockfile==3.0.post1
zipp==3.15.0
zipp @ file:///croot/zipp_1704206909481/work
zope-event==5.0
zope-interface==6.1
zstandard==0.22.0
```

</details>

--- a/src/bentoml/_internal/frameworks/transformers.py
+++ b/src/bentoml/_internal/frameworks/transformers.py
@@ -18,7 +18,7 @@ from ...exceptions import MissingDependencyException
 from ...exceptions import NotFound
 from ..models.model import Model
 from ..models.model import ModelContext
-from ..models.model import ModelOptions
+from ..models.model import ModelOptions as BaseModelOptions
 from ..models.model import ModelSignature
 from ..tag import Tag
 from ..types import LazyType
@@ -170,7 +170,7 @@ def _autoclass_converter(
 
 
 @attr.define
-class TransformersOptions(ModelOptions):
+class ModelOptions(BaseModelOptions):
     """Options for the Transformers model."""
 
     task: str = attr.field(factory=str, validator=attr.validators.instance_of(str))
@@ -240,7 +240,7 @@ class TransformersOptions(ModelOptions):
         return task_impl
 
     @classmethod
-    def from_task(cls, task: str, definition: TaskDefinition) -> TransformersOptions:
+    def from_task(cls, task: str, definition: TaskDefinition) -> ModelOptions:
         return cls(
             task=task,
             tf=definition.get("tf", None),  # type: ignore (handle by cattrs converter)
@@ -319,7 +319,7 @@ def register_pipeline(
     default: DefaultMapping | None = None,
     type: str | None = None,
 ):
-    task_impl = TransformersOptions.process_task_mapping(impl, pt, tf, default, type)
+    task_impl = ModelOptions.process_task_mapping(impl, pt, tf, default, type)
 
     if HAS_PIPELINE_REGISTRY:
         from transformers.pipelines import PIPELINE_REGISTRY
@@ -399,7 +399,7 @@ def load_model(bento_model: str | Tag | Model, *args: t.Any, **kwargs: t.Any) ->
             f"Model {bento_model.tag} was saved with module {bento_model.info.module}, not loading with {MODULE_NAME}."
         )
 
-    options = t.cast(TransformersOptions, bento_model.info.options)
+    options = t.cast(ModelOptions, bento_model.info.options)
     api_version = bento_model.info.api_version
     task = options.task
     pipeline: transformers.Pipeline | None = None
@@ -898,7 +898,7 @@ def import_model(
         api_version=API_VERSION,
         signatures=signatures,
         labels=labels,
-        options=TransformersOptions(),
+        options=ModelOptions(),
         custom_objects=custom_objects,
         external_modules=external_modules,
         metadata=metadata,
@@ -1107,7 +1107,7 @@ def save_model(
             api_version=API_VERSION,
             labels=labels,
             context=context,
-            options=TransformersOptions.from_task(*options_args),
+            options=ModelOptions.from_task(*options_args),
             signatures=signatures,
             custom_objects=custom_objects,
             external_modules=external_modules,
@@ -1150,7 +1150,7 @@ def save_model(
             api_version=API_VERSION,
             labels=labels,
             context=context,
-            options=TransformersOptions(),
+            options=ModelOptions(),
             signatures=signatures,
             custom_objects=custom_objects,
             external_modules=external_modules,

